CMakeLists.txt

```diff
@@ -42,6 +44,7 @@ set(QL_INSTALL_CMAKEDIR "lib/cmake/${PACKAGE_NAME}" CACHE STRING
 option(QL_BUILD_BENCHMARK "Build benchmark" ON)
 option(QL_BUILD_EXAMPLES "Build examples" ON)
 option(QL_BUILD_TEST_SUITE "Build test suite" ON)
+cmake_dependent_option(QL_BUILD_FUZZ_TEST_SUITE "Build fuzz test suite" OFF "'${CMAKE_CXX_COMPILER_ID}' MATCHES 'Clang'"  OFF)
```

Comment:
Thanks!  I think at this point this can just be
```
option(QL_BUILD_FUZZ_TEST_SUITE "Build fuzz test suite" OFF)
```
Right?

Reply:
It can be, although this will still likely fail to build if you are not using clang, as gcc doesn't support libfuzzer out of the box. The dependent_option will only make that option visible if the compiler is clang.

Reply:
Oh, I see.  I misinterpreted the semantics of `cmake_dependent_option`.